src/main/java/Command.java

```diff
@@ -0,0 +1,187 @@
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.List;
+
+public class Command {
+    private final String[] inputLine;
+    private boolean bye = false;
```

Comment:
Perhaps you could change the variable's name to isBye instead of bye since it is a boolean?